apps/question-bank/src/test/java/gov/cdc/nbs/questionbank/pagerules/PageRuleServiceImplTest.java

```diff
@@ -454,4 +455,49 @@ void shouldCreateJsForEnableAndDisableFunction() {
         Assert.assertEquals(jsonString, jsFunctionNameHelper.jsFunction());
     }
 
+    @Test
+    void getAllPageRuleTest() {
```

Comment:
The service is basically mapping a `WaRuleMetadata` to a `ViewRuleResponse` and that mapping is not being tested.